webpack/assets/javascripts/react_app/components/PasswordStrength/index.js

```diff
@@ -2,14 +2,15 @@ import { bindActionCreators } from 'redux';
 import { connect } from 'react-redux';
 
 import * as actions from './PasswordStrengthActions';
-import { doesPasswordsMatch } from './PasswordStrengthSelectors';
+import { doesPasswordsMatch, passwordPresent } from './PasswordStrengthSelectors';
 import reducer from './PasswordStrengthReducer';
 
 import PasswordStrength from './PasswordStrength';
 
 // map state to props
 const mapStateToProps = ({ passwordStrength }) => ({
   doesPasswordsMatch: doesPasswordsMatch(passwordStrength),
+  passwordPresent: passwordPresent(passwordStrength ? passwordStrength.password : ''),
```

Comment:
I don't think you need `passwrodPresent`, you can use your ternary condition and get the same results:

```js
'' ? true : false // => false
undefined ? true : false // => false
'some string' ? true : false // => true
```

Reply:
I think this logic should move to the selector itself.